coalib/parsing/Filter.py

```diff
@@ -0,0 +1,19 @@
+from coalib.parsing.FilterImplementation import *
+
+
+class Filter:
+    all_filters = {'language': LanguageFilter}
+
+    def __init__(self):
+        pass
+
+    def is_valid_filter(filter):
+        return filter in Filter.all_filters
+
+    @staticmethod
+    def get_filtered_bears(filter, args, log_printer):
```

Comment:
don't use `log_printer`, use `logging` instead. `log_printer` is inofficially deprecated and we are going to remove it step by step in future. The current CLI is just quite complicated and we are all to lazy to do that properly :/